com/win32com/__init__.py

```diff
@@ -104,8 +104,21 @@ def __PackageSupportBuildPath__(package_path):
 			__gen_path__ = os.path.join(
 								win32api.GetTempPath(), "gen_py",
 								"%d.%d" % (sys.version_info[0], sys.version_info[1]))
+	# Check if the user could write into the directory if not frozen, or otherwise later 
```

Comment:
I'm not that keen on this to be honest - it's extra write IO that often isn't necessary, and it doesn't seem safe from races between multiple processes? I wonder if we can make this while thing smarter and have the entire gen_py stuff be lazy? OTOH, in what circumstances will people hit this - if that dir exists but we don't use it, something else has probably gone wrong. Could you please revert this part, then I can land the rest of the PR, and we can work out what the best thing to do here is.

Reply:
Thanks that you spotted this. This should not have been part of the PR. I'll remove this.

Reply:
Change reverted.